cmd/clusterctl/clusterdeployer/clusterdeployer_test.go

```diff
@@ -20,16 +20,27 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"strings"
 	"testing"
 
 	"github.com/pkg/errors"
 	apiv1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Seems like a duplicate import of the one above

Reply:
Fixed, thanks!